git_repo/services/ext/gitlab.py

```diff
@@ -16,8 +16,8 @@ class GitlabService(RepositoryService):
     fqdn = 'gitlab.com'
 
     def __init__(self, *args, **kwarg):
-        self.gl = gitlab.Gitlab(self.url_ro)
-        super(GitlabService, self).__init__(*args, **kwarg)
+        super().__init__(*args, **kwarg)
+        self.gl = gitlab.Gitlab(self.url_ro, ssl_verify=not self.insecure)
```

Comment:
I'd add a warning here to warn that we're using an insecure connection:

``` python
self.log.warn('⚠️ Using insecure HTTPS connection to {}.'.format(self.url_ro))
```

except if `gitlab.Gitlab`, `requests` or `httplib` is already shouting one out.